storage/index/block.go

```diff
@@ -111,86 +108,74 @@ func (b *block) EndTime() time.Time {
 	return b.endTime
 }
 
-func (b *block) updateWriteBatchWithLock(inserts []WriteBatchEntry) {
-	for _, insert := range inserts {
```

Comment:
the new version of `WriteBatchEntry` has idents, not docs. so when you revert, this will have to do the convert.FromMetric call